docs/Manuals/Server/options.rst

```diff
@@ -0,0 +1,1176 @@
+  to a maximum value of two-thirds of ``timeout``. ``unitwaittime`` (aka UWT) is a tool to help reduce
+  :term:`RTS` around turn change (:term:`TC`) in Longturn games.
+
+``unitwaittime_extended``
+  :strong:`Value (Min, Max)`: disabled
+
+  :strong:`Description`: ``unitwaittime`` also applies to newly-built and captured/bribed units. If set,
+  newly-built units are subject to ``unitwaittime`` so that the moment the city production was last touched
+  counts as their last *action*. Also, getting captured/bribed counts as action for the victim.
+
+``unitwaittime_style``
+  :strong:`Value (Min, Max)`: empty value/not set
+
+  :strong:`Description`: This setting affects ``unitwaittime`` and effectively unused as it only has one option
+  to set:
+
+    * Activities (``ACTIVITIES``): Units moved less than ``unitwaittime`` seconds from turn change will not
+      complete activities such as pillaging and building roads during turn change, but during the next turn
+      when their wait expires.
+
+``unreachableprotects``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Does unreachable unit protect reachable ones. This option controls whether tiles with
+  both unreachable and reachable units can be attacked. If disabled, any tile with reachable units can be
+  attacked. If enabled, tiles with an unreachable unit in them cannot be attacked. Some units in some rulesets
+  may override this, never protecting reachable units on their tile.
+
+``victories``
+  :strong:`Value (Min, Max)`: Spacerace (``SPACERACE``) and Allied victory (``ALLIED``)
+
+  :strong:`Description`: What kinds of victories are possible for the game. This setting controls how a game
+  can be won. One can always win by conquering the entire planet, but other victory conditions can be enabled
+  or disabled:
+
+    * Spacerace (``SPACERACE``): Spaceship is built and travels to Alpha Centauri.
+    * Allied (``ALLIED``): After defeating enemies, all remaining players are allied.
+    * Culture (``CULTURE``): Player meets ruleset defined cultural domination criteria.
+
+``wetness``
+  :strong:`Value (Min, Max)`: 50 (0, 100)
+
+  :strong:`Description`: Amount of water on the landmasses. Small values mean lots of dry, desert-like land.
+  Higher values give a wetter map with more swamps, jungles, and rivers.
+
+``xsize``
+  :strong:`Value (Min, Max)`: 64 (16, 128000)
+
+  :strong:`Description`: Map width in tiles. Defines the map width. For this option to take effect, the
+  "Map size definition" option (``mapsize``) must be set to "Width and height" (``XYSIZE``).
+
+``ysize``
+  :strong:`Value (Min, Max)`: 64 (16, 128000)
+
+  :strong:`Description`: Map height in tiles. Defines the map height. For this option to take effect, the
+  "Map size definition" option (``mapsize``) must be set to "Width and height" (``XYSIZE``).
+
+  .. raw:: html
+
+        <p>&nbsp;</p>
```

Comment:
Why is this here? It doesn't appear in other files

Reply:
the original file was a giant table and i added that to create some whitespace between the table and the buttons at the bottom of the page.